src/app.ts

```diff
@@ -154,6 +157,10 @@ async function createApp(): Promise<express.Application> {
 
   app.use(loggerMiddleware);
   app.use(healthcheckRouter);
+  if (getNodeEnv() === "staging") {
```

Comment:
let's check this is the right config variable

Reply:
Please could you use getAppEnv() instead? node env is a node thing rather than our environment name.  Have a look in the ecs task definition in a deployed environment to see what these are set to.